### DISCUSSION TITLE
请问下自动签到在哪里体现

### DISCUSSION THREAD

**Discussion by lifesign** [qier222/YesPlayMusic#1344] [Q&A]
看到功能特性中有自动签到，但是打开客户端并没有任何提示，是否可以显示一下签到状态，万一签到失败是不是也可以提供手动补签功能

**Comment by pan93412** [comment]
啟動 app 時會自動呼叫 NCM 簽到的 API。

**Reply by gov360** [reply]
> 啟動 app 時會自動呼叫 NCM 簽到的 API。

连续多天不关闭程序时，是否也能每天自动签到？
我的系统是linuxmint 20.3 软件版本0.4.5
